pkg/mysql/server.go

```diff
@@ -183,7 +183,10 @@ func (l *Listener) handle(conn net.Conn, connectionID uint32) {
 		c.sequence = 0
 		data, err := c.readEphemeralPacket()
 		if err != nil {
-			c.recycleReadPacket()
+			// Don't log EOF errors. They cause too much spam.
```

Comment:
must call recycleReadPacket.

Reply:
if do not call recycleReadPacket, what will happen？

Reply:
ok, when there is an error, will close the connection, then recycleReadPacket is not necessary.